### ISSUE TITLE
Build and push images immediately, even when waiting on resource dependencies

### ISSUE THREAD

**Issue by nathanperkins** [tilt-dev/tilt#5962]
## Describe the Feature You Want
I want images associated with k8s resources to build and push immediately, regardless of whether they are waiting on dependencies.

### Current Behavior
Images only start to build and push once dependent resources are ready.

### Why Do You Want This?
Building and pushing images as soon as possible would significantly reduce the amount of time it takes to `tilt up`.

**Comment by nicks** [comment]
thanks for the report! I'm going to mark this as a duplicate of https://github.com/tilt-dev/tilt/issues/2770
